### PR TITLE
Add Kebechet tooling section

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/AICoE/data-driven-development/issues/35

### CODE REVIEW

File: examples/kebechet/tooling.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    - [Kebechet Administrator Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet-administrator.yaml) - run Kebechet administrator that decides based on triggers which repo needs to receive a Kebechet update.
+    - [Adviser Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/adviser/base/openshift-templates/adviser.yaml) - run Thoth adviser recommender system.
+    - [Kebechet Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet.yaml) - run Kebechet component logic.
+    - [Solver Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/solver/base/openshift-templates/solve_and_sync-template.yaml) - run solver component to analyze python packages for different runtime environments, stores knowledge in the graph database and sends messages for scheduling Security workflows.
+
+
+### Monitoring Components
+
+* [Prometheus](https://prometheus.io/) - Monitoring tool for collecting time series metrics.
```
Comment (hemajv) [inline examples/kebechet/tooling.md:41]: Can you also please add "To learn more about Prometheus, refer to this documentation [here](https://github.com/AICoE/data-driven-development/blob/main/data-sources/prometheus.md)."

Comment (pacospace) [reply examples/kebechet/tooling.md:41]: Nice! yes :)

File: examples/kebechet/tooling.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* [Grafana](https://grafana.com/) - Visualization tool for creating dashboards/graphs of the metrics.
+* [Thoth MI Scheduler](https://github.com/thoth-station/mi-scheduler) - Tool to collect GitHub data.
+* [GitHub Alertmanager](https://github.com/prometheus/alertmanager) - Alertmanager that handles alerts and creates GitHub issues in the chosen repository to notify the respective teams.
+* [Thoth Metrics Exporter](https://github.com/thoth-station/metrics-exporter) - Component that evaluates content metrics from the database and sends metrics to Prometheus.
+* [Thoth Graph metrics exporter](https://github.com/thoth-station/graph-metrics-exporter) - Component that evaluates content metrics for long running processes (evaluating bloat data for example) from the database and sends metrics to Prometheus.
+* [PostgreSQL metrics exporter](https://github.com/prometheus-community/postgres_exporter) - Components that collect operational metrics from the database and send them to Prometheus.
+
+
+### Analytical Components
```
[the remark this answers is not in the document]
Comment (pacospace) [reply examples/kebechet/tooling.md:50]: Yes we should! Thanks!

File: examples/kebechet/tooling.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- `application` -> components or libraries that are created for the application
+
+- `monitoring` -> components or libraries that are created/used for monitoring and pushing metrics
+
+- `analytics` -> components or libraries that are created/used for providing analysis on the data collected
+
+
+### Service components
+
+* [Kafka](https://kafka.apache.org/) - an open-source distributed event streaming platform.
```
[the remark this answers is not in the document]
Comment (pacospace) [reply examples/kebechet/tooling.md:20]: Sure!

**Comment by hemajv** [comment]
great! :100: 

/approve

**Pull request merged**